src/app/public/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
-    "short_name": "React App",
-    "name": "Create React App Sample",
+    "short_name": "Fishway",
+    "name": "Fishway",
```

Comment:
:tada: I don't think anyone will ever see this at the museum, but it was bugging me...